apps/mobile/src/components/draggable.tsx

```diff
@@ -0,0 +1,127 @@
+import { ReactNode, RefObject, useRef } from 'react';
+import { ScrollView } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, {
+  SharedValue,
+  runOnJS,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+const APPROXIMATE_DEFAULT_ITEM_SIZE = { height: 150, width: 300 };
+export function Draggable({
+  idx,
+  accountId,
+  accountsLength,
+  placeholderIdx,
+  scrollViewRef,
+  direction,
+  onAccountPress,
+  swapAccountIndexes,
+  Component,
+}: {
+  idx: number;
+  accountId: string;
+  accountsLength: number;
```

Comment:
This component being called `Draggable` makes me think it's a generic component that can be used elsewhere. But it can't, it's specific to the account re-ordering use-case.

Suggest either refactoring to be reusable, or renaming and relocating to the feature folder

Reply:
I was trying to get him to refactor it so it was reusable, but I overlooked there were still account specific prop names. My thought was we will have other uses for the draggable code and it should be generic?